Packs/ServiceNow/Integrations/ServiceNowEventCollector/ServiceNowEventCollector_description.md

```diff
@@ -0,0 +1,17 @@
+Use this integration to collect audit logs automatically from ServiceNow.
+
+To use ServiceNow on Cortex XSIAM, ensure your user account has the rest_api_explorer and web_service_admin roles.
+These roles are required to make API calls.
+However, they may not suffice for viewing records in some tables.
+Please make sure you have the correct role so you have permissions to work with the relevant table.
```

Comment:
How does the user know which roles he needs to view some tables?

Reply:
This is an internal ServiceNow role management. If as a user of ServiceNow he has the permissions to work with the relevant table, he will be able to fetch in XSIAM as well